calendar/command/command.go

```diff
@@ -54,7 +54,7 @@ var cmds = []*model.AutocompleteData{
 			model.NewAutocompleteData("disable", "", "Disable your daily summary."),
 		},
 	},
-	model.NewAutocompleteData("viewcal", "", "View your events for the upcoming week."),
+	model.NewAutocompleteData("viewcal", "", "View your events for the current and upcoming week."),
```

Comment:
@Kshitij-Katiyar  If I ran this command on a Wednesday, would it show events for the Wednesday, Thursday, Friday, and every day the week after?

Reply:
@wiggin77 Please refer to this comment for more clarity: https://github.com/mattermost/mattermost-plugin-mscalendar/issues/392#issuecomment-2293011803